include/rcutils/macros.h

```diff
@@ -57,7 +57,12 @@ extern "C"
   #endif
 #else
 // Some other non-Windows, non-cygwin, non-apple OS
-  #define RCUTILS_THREAD_LOCAL _Thread_local
+  #if  __STDC_VERSION__ >= 201112L
+// C11 enabled
```

Comment:
Please move this comment at the end of the previous line (or indent it to the level of the current block).